src/logic/operate.js

```diff
@@ -0,0 +1,37 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation ) => {
+  numberOne = new Big(numberOne);
+  numberTwo = new Big(numberTwo);
+
+  switch (operation) {
+    case "+":
+      return numberOne.plus(numberTwo).toString();
+      break;
+
+    case "-":
+      return numberOne.minus(numberTwo).toString();
+      break;
+
+    case "X":
+      return numberOne.times(numberTwo).toString();
+      break;
```

Comment:
### NO need for `break` after a `return` statement -> The function won't even see any of the `break` statement after it `returns`